.github/workflows/deploy-preview-storefrontcloud.yml

```diff
@@ -28,14 +28,25 @@ jobs:
           ../packages/cli/bin/shopware-pwa init --shopwareEndpoint=${{ env.CANARY_SW6_INSTANCE }} --shopwareAccessToken=${{ env.CANARY_SW6_INSTANCE_TOKEN }} --u ${{ secrets.SHOPWARE_ADMIN_USER }} --p ${{ secrets.SHOPWARE_ADMIN_PASSWORD }} --ci --devMode --stage local
           yarn build
       - name: Build and publish docker image
-        uses: elgohr/Publish-Docker-Github-Action@master
+        uses: elgohr/Publish-Docker-Github-Action@2.14
+        if: github.repository_owner	== 'DivanteLtd'
         with:
           name: shopware-pwa-storefrontcloud-io/vue-storefront:${{ github.sha }}
           registry: registry.storefrontcloud.io
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
           workdir: ./test-project
           buildoptions: "--compress"
+      - name: Add information about PR from forked repo
+        if: github.repository_owner	!= 'DivanteLtd'
+        uses: mshick/add-pr-comment@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          allow-repeats: false
+          message: |
+            :warning: This PR comes from forked repository and cannot be deployed on our preview instance!
```

Comment:
@patzick here we comment the PR with the appropriate message in case of PR - temporary until we have it moved to the circleCi